src/notte/utils/code.py

```diff
@@ -0,0 +1,22 @@
+def has_indentation(text: str) -> bool:
```

Comment:
a clearer name would be `text_contains_tabs` indentation could mean different things. We